frontend/lib/framework/error-classes-list.cpp

```diff
@@ -128,6 +129,68 @@ void ErrorMissingInclude::write(ErrorWriterBase& wr) const {
   wr.note(moduleInclude, "expected file at path '", filePath, "'");
 }
 
+void ErrorUseImportUnknownSym::write(ErrorWriterBase& wr) const {
+  auto visibilityClause = std::get<const uast::VisibilityClause*>(info);
+  auto symbolName = std::get<std::string>(info);
+  auto searchedScope = std::get<const resolution::Scope*>(info);
+  auto useOrImport = std::get<const resolution::VisibilityStmtKind>(info);
+  wr.heading(kind_, type_, locationOnly(visibilityClause),
+             "cannot find symbol '", symbolName, "' for ", useOrImport, ".");
+  wr.message("Symbol named here:");
+  wr.code(visibilityClause);
+  wr.message("Searching in scope of '", searchedScope->name(), "':");
+  wr.code(searchedScope->id());
+}
+
+void ErrorUseImportUnknownMod::write(ErrorWriterBase& wr) const {
+  auto id = std::get<const ID>(info);
+  auto moduleName = std::get<std::string>(info);
+  auto useOrImport = std::get<const resolution::VisibilityStmtKind>(info);
+  wr.heading(kind_, type_, id, "cannot find module '", moduleName,
+             "' for ", useOrImport, ".");
+  wr.message("Module named here:");
+  wr.code(id);
```

Comment:
It looks good (image because it is not pasting well):
![image](https://user-images.githubusercontent.com/14242898/197589230-f696953f-9008-4d4e-8854-27c9b62a0a4d.png)

Going with this.